src/ui/actions/eventListeners/constants.ts

```diff
@@ -0,0 +1,15 @@
+export type InteractionEventKind = "mousedown" | "keypress";
+
+export const REACT_EVENT_PROPS: Record<InteractionEventKind, string[]> = {
+  mousedown: ["onClick"],
```

Comment:
What about `"onMouseDown"` or `"onMouseUp"`?

Reply:
Intentionally not handling "down/up" events for the event listeners / jump to code logic.  We don't show those in the info sidebar or the Cypress panel - it's just "clicks" and "keypresses".

Reply:
Why?

Reply:
That won't actually work given the current implementation.  The sidebar list items are _specifically_ the timestamps + points for just `click` and `keypress` events, so if we start trying to add the up/down events, the timestamps won't match up.

Reply:
Yep - we'd have to improve the UI to also include other click-like events as well and rollup events that are close together.

Reply:
Yeah, and per discussion in Discord, I'm not sure that makes sense "conceptually" given that the list is currently representing "times that the user interacted with the app" (and _not_ "individual sub-events for one overall interaction")

Reply:
The things you call "sub events" are user interactions though.

If we're trying to help people find where their code handled mouse or keyboard events, then not including these commonly used event types seems like an ommission.

Reply:
If we want to have a larger design discussion around changing all this, we can. But everything I've built so far for "Jump to Code" / "Event Listeners" has been based around the original design and behavior of the info sidebar.  Arguing about it in this PR (which is just a refactoring PR in the first place) doesn't really seem helpful or relevant :(

Reply:
FE-1496

Reply:
Hey gang, just reading up on this.

I agree with and sympathise with the idea that commonly used event types that don't show up can feel confusing. Coming up with ways to message that, even if it's just mentioned in docs, is better than nothing.

I am glad we have filed FE-1496 to split that conversation into its own thing.

At all times we need to remind ourselves that the goal is "better than before." Jump to Code is 100% better than not having it. And I assume this refactor is better than life before the refactor.

We can discuss in FE-1496 what our new "better than before" bar includes, and if it's worth the time commitment, etc.